### ISSUE TITLE
Spinestalker & Rimefang - missing immunities

### ISSUE THREAD

**Issue by bLeyaa** [WoWManiaUK/Redemption#4281]
**Links:** [Spinestalker](https://www.wowhead.com/npc=37534/spinestalker) & [Rimefang](https://www.wowhead.com/npc=37533/rimefang)

**What is Happening:** They currently have no immunity to any of the mechanics except for Fear.

**What Should happen:** Both of the mobs should be immune to the below mechanics:

MECHANIC_CHARM	
MECHANIC_DISORIENTED	
MECHANIC_DISTRACT	
MECHANIC_FEAR	
MECHANIC_GRIP	
MECHANIC_ROOT	
MECHANIC_SLOW_ATTACK	
MECHANIC_SLEEP	
MECHANIC_SNARE	
MECHANIC_STUN	
MECHANIC_FREEZE	
MECHANIC_KNOCKOUT	
MECHANIC_SHACKLE	
MECHANIC_TURN	
MECHANIC_DAZE

**Comment by Rushor** [comment]
will add that.

**Comment by tvshadowz** [comment]
I can confirm that they can be shackled stunned etc.

**Comment by Rushor** [comment]
applied for all difficuluty modes for the 2 creatures

**Comment by tvshadowz** [comment]
Will test in the morning

**Comment by bLeyaa** [comment]
Immunities are there, the only one missing if for the Thunderstorm (ID 51490).

**Comment by Rushor** [comment]
knockback is handled in flag_extra - new ticket for that maybe in future

**Comment by bLeyaa** [comment]
@jeyrella @Rushor @tvshadowz Both of them can now be cycloned again. People abuse Rimefang air phase by doing cyclone as soon as he starts moving up so once the cyclone is done he remains in the spot where he was before cyclone.

**Comment by Rushor** [comment]
should be immune again
